scripts/test-oidc-pages.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Do we still need this? looks like there is no more vars in the file.

Reply:
`no-var-requires` is about converting `require()` to `import` or `import()`: https://typescript-eslint.io/rules/no-var-requires/

But that doesn't work for plain `.js` scripts